apps/server/src/testing/test-config.helper.ts

```diff
@@ -0,0 +1,20 @@
+export class TestConfigHelper<T, K extends keyof T = Extract<keyof T, string>> {
```

Comment:
I'm not sure about the name suffix `Helper`. Can we just call the class `TestConfig`?